### PR TITLE
added acronyms to handbook

### PR DESCRIPTION
**Pull request by andreas-kater**

### CODE REVIEW

File: guides/acronyms.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+The below is a list of acronyms that are commonly used in the work we do, along
+with their definitions. If you come across an acronym you don't know this is a
+good place to reference.
+
+| Acronym | Definition |
+| --- | --- |
+| AGM | Annual General Meeting |
+| ASR | Automated Speech Recognition |
+| Autopilot | Twilio Product? |
```
Comment (tom-mullen) [inline guides/acronyms.md]: This is both a Twilio and DVELP project.

File: guides/acronyms.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| BAU | Business as usual |
+| BCM | Business Continuity Management |
+| canned responses | pre-determined and saved response, ready to be called programatically. E.g. "Please hold...|
```
Comment (tom-mullen) [inline guides/acronyms.md]: Titlecase?

File: guides/acronyms.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+| CLI | Command line interface |
+| DBS | Disclosure and Barring Service (Police background check) |
+| DDI | Direct Dial-In, dedicated number (used in the background, not linked to MPOA) |
+| DDOS | Distributed Denial of Service Attack |
+| DOS | Denial of service attack |
+| DPIA | Data Protection Impact Assessment |
+| DPIA | Data Protection Impact Assessments |
```
Comment (tom-mullen) [inline guides/acronyms.md]: Dupe

File: guides/acronyms.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+| DPO | Data Protection Officer |
+| DTMF | Pressing button on dialphone to direct call during call (Dual-tone multi-frequency signaling) |
```
Comment (tom-mullen) [inline guides/acronyms.md]: Doesn't read very well

File: guides/acronyms.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+| E164 | aka long number e.g. +447500172268 |
```
Comment (tom-mullen) [inline guides/acronyms.md]: Title case

File: guides/acronyms.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+| FOIA | Freedom of Information Act |
+| GDPR | General Data Protection Regulation |
+| GTM | Go to market |
+| ICO | Information Commissioner's Office |
+| IEC | International Electrotechnical Commission (Standardisation body linked to ISO) |
+| ISMS | Information Security Management System |
+| ISO | International Organization for Standardization |
+| ISV  | Independent Software Vendor |
```
Comment (tom-mullen) [inline guides/acronyms.md]: Extra space in the table cell

File: guides/acronyms.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+| IVR | Interactive Voice Response |
+| MPOA | Publicly displayed number |
+| MQL | Marketing Qualified Lead |
+| MSA | Master Service Agreement |
+| Multi-modal | e.g. starting on phone, using SMS/email/app to comlete a process |
```
Comment (tom-mullen) [inline guides/acronyms.md]: Typo

File: guides/acronyms.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+| NDA | Non-disclosure Agreement |
+| NLR | Natural Language Recognition |
+| OWASP | Open Web Application Security Project www.owasp.org |
+| PBX | Private Branch Exchange, which is a private telephone network used within a company or organization to switch calls internally |
+| PCI DSS | Payment Card Industry Data Security Standard  |
+| PDP | Personal Development Programme |
+| Predictive dialer | A predictive dialer dials a list of telephone numbers and connects answered dials to people making calls, often referred to as agents. |
```
Comment (tom-mullen) [inline guides/acronyms.md]: Title case. Also, I think this is no longer the correct definition, given what we learned?

File: guides/acronyms.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+| PSTN | Public Switched Telephone Network |
+| RTC | Real Time Communications |
+| SIP | Session Initiation Protocol |
+| SMB | Small to medium sized business |
+| SSH | Secure Shell (tool for accessing a server remotely via the command line) |
+| SSL | Secure Sockets Layer (depricated, now referred to as TLS or Transport Layer Security) |
+| SSO | Single Sign On  |
+| TSL | Transport Layer Security  |
```
Comment (tom-mullen) [inline guides/acronyms.md]: TSL -> TLS

File: guides/acronyms.md (hunk 8 above)
Comment (andreas-kater) [reply guides/acronyms.md]: Good point. Do you agree with the new definition?

**Review by tom-mullen — APPROVED** [review]

**Pull request merged**